src/app/App/component.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
-import Container from 'react-bootstrap/Container';
+import Header from '../../shared/components/Header/index';
```

Comment:
we could just import without index file mentioned in path - it will be looking for `index.ts` file by default.

```
import Header from 'shared/components/Header;
```